teachers.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As I mentioned above, you need to pass the arguments properly to make sure that your application should work fine without any errors. Please take the name and age argument in the constructor. For the `parent_permission`. It is understandable that `parent_permission` has a default value of true in case `nothing` is sent explicitly.